### PR TITLE
MAINT: Refresh cached intermediate results

### PR DESCRIPTION
**Pull request by oesteban**
We are seeing some metrics changing in PRs, while master does not seem to bother about this. This PR investigates these changes and will finally update the metrics.

cc/ @arokem we were talking about this yesterday.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -397,7 +397,7 @@ jobs:
           path: /tmp/t1w/misc
 
       - save_cache:
-          key: t1w-v5-{{ epoch }}
+          key: t1w-v6-{{ epoch }}
```
Comment (effigies) [inline .circleci/config.yml:400]: If this isn't really supposed to change, I would not pin it to the epoch. Just pin it to the branch.

```suggestion
          key: t1w-v6-{{ .Branch }}
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -359,8 +359,8 @@ jobs:
             - data-v2-
       - restore_cache:
           keys:
-            - t1w-v5-{{ epoch }}
-            - t1w-v5-
+            - t1w-v6-{{ epoch }}
+            - t1w-v6-
```
Comment (effigies) [inline .circleci/config.yml:363]:
```suggestion
            - t1w-v6-{{ .Branch }}
            - t1w-v6-master
            - t1w-v6-
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -519,8 +519,8 @@ jobs:
             - data-v2-
       - restore_cache:
           keys:
-            - bold-v4-{{ epoch }}
-            - bold-v4-
+            - bold-v5-{{ epoch }}
+            - bold-v5-
```
Comment (effigies) [inline .circleci/config.yml:523]:
```suggestion
            - bold-v5-{{ .Branch }}
            - bold-v5-master
            - bold-v5-
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -555,7 +555,7 @@ jobs:
           path: /tmp/bold/misc
 
       - save_cache:
-          key: bold-v4-{{ epoch }}
+          key: bold-v5-{{ epoch }}
```
Comment (effigies) [inline .circleci/config.yml:558]:
```suggestion
          key: bold-v5-{{ .Branch }}
```

File: .circleci/config.yml (hunk 1 above)
Comment (oesteban) [reply .circleci/config.yml:400]: I agree - I'll accept the changes after the PR breaks on Circle

**Review by effigies — APPROVED** [review]
Done been fixed.

**Pull request merged**